lib/config/hse_gparams.c

```diff
@@ -192,18 +177,18 @@ static const struct param_spec pspecs[] = {
         .ps_flags = 0,
         .ps_type = PARAM_TYPE_ENUM,
         .ps_offset = offsetof(struct hse_gparams, gp_logging.level),
-        .ps_size = sizeof(hse_logpri_t),
+        .ps_size = sizeof(enum log_priority),
         .ps_convert = param_default_converter,
         .ps_validate = param_default_validator,
         .ps_stringify = param_default_stringify,
         .ps_jsonify = param_default_jsonify,
         .ps_default_value = {
-            .as_scalar = HSE_LOGPRI_DEFAULT,
+            .as_scalar = LP_DEFAULT,
         },
         .ps_bounds = {
             .as_scalar = {
-                .ps_min = HSE_LOGPRI_EMERG,
-                .ps_max = HSE_LOGPRI_DEBUG,
+                .ps_min = LP_MIN,
+                .ps_max = LP_MAX,
```

Comment:
Line Printer?  LOGPRI so I don't have to guess..  You keep removing the namespace HSE_ from things, that can bite you if you define your macros and then include other files that can't ensure don't use or redefine that name..

Reply:
We can tackle name collisions when we get there. There aren't any now, so I'm not worried about it.

We are good about hidden symbol visibility too.

I will change the identifier name though. I mostly just did it to match "LD_" above